dlrm_s_pytorch.py

```diff
@@ -295,7 +308,9 @@ def interact_features(self, x, ly):
         return R
 
     def forward(self, dense_x, lS_o, lS_i):
-        if self.ndevices <= 1:
+        if ext_dist.my_size > 1:
+            return self.distributed_forward(dense_x, lS_o, lS_i)
+        elif self.ndevices <= 1:
             return self.sequential_forward(dense_x, lS_o, lS_i)
         else:
```

Comment:
Is this still needed ? wouldn't distributed_forward also work for the single node case ?

Reply:
For single rank, it still uses `sequential_forward` to avoid any overheads for multiranks.

Reply:
my bad, I was referring to line#316 - with using parallel_forward, wouldn't distributed forward cover that case as well ?

Reply:
No, if you want to use multi-gpu (with single process), it will still use `parallel_forward`. `distributed_forward` only support single gpu per rank.